<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>912</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1162</width>
    <height>912</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1162</width>
    <height>912</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ConverterPIX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="input_tabs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>1121</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="model_tab">
     <attribute name="title">
      <string>Model</string>
     </attribute>
     <widget class="QListWidget" name="model_paths">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>471</width>
        <height>311</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="model_relative_path">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QLabel" name="model_relative_path_lbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>12</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Relative path:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>471</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="flow">
       <enum>QListView::LeftToRight</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="animation_selected">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>30</y>
        <width>221</width>
        <height>341</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="animations_selected_lbl">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Selected animations:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="animation_g_path">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>30</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="flow">
       <enum>QListView::LeftToRight</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="animation_paths">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>60</y>
        <width>271</width>
        <height>311</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="animations_paths_lbl">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select animations:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="animation_remove_btn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="animation_add_btn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="animation_relative_path">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>370</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="texture_tab">
     <attribute name="title">
      <string>Texture</string>
     </attribute>
     <widget class="QListWidget" name="texture_paths">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>1071</width>
        <height>311</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="texture_relative_path_lbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>12</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Relative path:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="texture_relative_path">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>1071</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QListWidget" name="texture_relative_grapich">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>1071</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="flow">
       <enum>QListView::LeftToRight</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ConverterPIX</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ufs_paths_lbl">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>61</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Base paths: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="ufs_add_btn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ufs_paths">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ufs_unmount_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Unmount</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ufs_up_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ufs_down_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>157</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="convert_btn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>140</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert selected items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_path_btn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>199</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="export_path">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="export_path_lbl">
    <property name="geometry">
     <rect>
      <x>341</x>
      <y>185</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Export path:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="model_paths_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>1121</width>
      <height>201</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="log_label">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>671</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Log:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convert_models_btn">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>80</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert all models</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ufs_paths</tabstop>
  <tabstop>ufs_add_btn</tabstop>
  <tabstop>ufs_unmount_btn</tabstop>
  <tabstop>ufs_up_btn</tabstop>
  <tabstop>ufs_down_btn</tabstop>
  <tabstop>input_tabs</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>model_paths</tabstop>
  <tabstop>model_relative_path</tabstop>
  <tabstop>animation_g_path</tabstop>
  <tabstop>animation_add_btn</tabstop>
  <tabstop>animation_paths</tabstop>
  <tabstop>animation_remove_btn</tabstop>
  <tabstop>animation_selected</tabstop>
  <tabstop>animation_relative_path</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
